source/client/client.js

```diff
@@ -0,0 +1,90 @@
+function render(gameState) {
+  $('#grid').empty();
+  for (let i = 0; i < gameState.board.length; i++) {
+    const row = $('<div />')
+      .addClass('row')
+      .attr('id', `row-${i}`);
+    // place row in grid
+    $(grid).append(row);
+    for (let j = 0; j < gameState.board[0].length; j++) {
+      const column = $('<div />')
+        .addClass('column')
+        .attr('id', `row-${i}-column-${j}`);
+      const circle = $('<div />')
+        .addClass('circle')
+        .attr('id', `circle-row-${i}-column-${j}`);
+      $(column).append(circle);
+      // place column in row
+      $(row).append(column);
+      if (gameState.board[i][j] !== null) {
+        console.log(gameState.board[i][j]);
+        $(`#circle-row-${i}-column-${j}`).css('background-color', gameState.board[i][j]);
+      }
+      $(column).click(() => $(() => {
```

Comment:
This extra layer of making JQuery execute this inner function is a bit unusual. Does it serve some purpose?
```suggestion
      $(column).click(() => {
```